cli/container.go

```diff
@@ -19,6 +22,17 @@ func (c *container) config() *types.ContainerCreateConfig {
 	// TODO
 	config.Tty = c.tty
 
+	// set labels
+	config.Labels = make(map[string]string)
+	for _, label := range c.labels {
+		fields := strings.SplitN(label, "=", 2)
+		if len(fields) != 2 {
+			continue
```

Comment:
I suggest that we should throw out the validation error message. If no error, user will by default consider his operation was right, but that is not true. 
So I think in cli, we should should report error as soon as possible.